src/schema/v2/partner/updatePartnerFlagsMutation.ts

```diff
@@ -0,0 +1,46 @@
+import { GraphQLString, GraphQLNonNull } from "graphql"
+import { mutationWithClientMutationId } from "graphql-relay"
+import { ResolverContext } from "types/graphql"
+import Partner from "./partner"
+
+interface UpdatePartnerFlagsMutationInputProps {
+  id: string
+}
+
+export const updatePartnerFlagsMutation = mutationWithClientMutationId<
```

Comment:
I'm not sure what this mutation has to do with CMS conversations rewrite?

Reply:
Following the guide linked above